detection_rules/devtools.py

```diff
@@ -732,6 +738,61 @@ def update_schemas():
         cls.save_schema()
 
 
+@dev_group.command('update-navigator-gists')
+@click.option('--directory', type=Path, default=CURRENT_RELEASE_PATH.joinpath('extras', 'navigator_layers'),
+              help='Directory containing only navigator files.')
+@click.option('--token', required=True, prompt=get_github_token() is None, default=get_github_token(),
```

Comment:
Looks like it passes a 401 which should be descriptive enough, especially since this is an internal only function (under `dev`)

```
401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest"}
```

Reply:
The error I received was a 404, which may have occurred when trying to use your gist_id. The error went away after using my own gist_id.

Reply:
ah, so 404 if authed but no gist_id associated with that user. We can catch this and suggest to validate gist_id exists and user has access 👍

Reply:
cd36a83